### ISSUE TITLE
[Feature Request] For Parameter Type=String with values=[list]

### ISSUE THREAD

**Issue by maybites** [ossia/score#1047]
Hi

If there is a parameter with the type=String with a range of multiple values,  for setting the 'Value' it would be nice if there was additionally a pull down menu to select from the range-values (like for selecting the Type - see screenshot), instead of entering the string. 
Ideally it would be possible to do both - depending of the Bounding/Clip setting: if set to clip/both you can only select, if set to free you also can enter your own value.

This would allow to seamlessly link selections between remote applications and score and reduce the likelihood of typos. (see also the patch below)

<img width="227" alt="Screenshot 2020-06-21 at 10 45 14" src="https://user-images.githubusercontent.com/148945/85220518-52049400-b3ac-11ea-9704-8f25e12df615.png">

<pre><code>
----------begin_max5_patcher----------
1676.3ocwYkzbaaCE9rzuBL5RtnpRvc1SNS5odsoWZlLZ.IgkPBIfJAnWRl7
euXg6hThVR19fkodOB7v2aGO8ykKVEydByWA9CvW.KV7ykKVnIoHrn56KVki
dJICw0u1JJ9QV72Vs1vRfeRnI+irME3GpIe.IR1Sn61VfSDlc2wIZi0Z.LBp
9mSf5Sa6MVfuVsFVoHCKDOe.aVvpUqk+0vljpkiT1+VTsXnk4DpbQ5SlcKQy
VYnpH9qkKUer95f3gBBUbJD5aq.kuuFf1GAvNHvcTD.GEAVW.BxwbNZG9HHj
jgQEm.BtdlisqmFCdyvHMF77dAFH3sy.kwPowH5tS.P6H2VqiW34wmd+FCi1
u.SH7V5DhOfoo.zA0+Ne7VkoLx+BMkv2IbJ.wfLv47WcrszIU7cz1yfYZOWa
RtX18wPc3K.zN2NPKShRD3B.bF1UnFrtWSdT+2m7nExsECJjFB7LLsNZO2.q
q.mAux3bprs6vhyBypDRlhFdQWZTp0qbF2xbLs77vvNDtIvwE5CcB7i7bf1d
xbOFT4bJToqrdjITfyq5MY0+hKXJtqUe7Oz1m+bgr+k5u72Dr.29MLok0exZ
e9SkHQms6SDZBqUr2ynh6QI5il0H0tca0CEnbo.K1hon3rdKPsIbxOzzfprS
yOeRmcfJ2esf+XAAks51ECx3bBR1wVNSFJxKiyYo3remKJTlUMtNuwFZYJpX
4OyXydF2DVFqvv0ZizOwMZs7gfHeqfP0SgQtPn7I33cPAguxoni0.G2T84AB
9wGHbRLIiHdtqrX2eOGKpghk5v2djyXIeGmlVf1wSJXYYc8QRxHIeWrWlOb2
9tzMdS6OdAFFObLi3cJCX2P9XVQJtn66LZ6d5lgbfarGFz5UULEFtIv2KzIz
21wx0OLvaTqgSsZBmGiSGXIZMOVi2ga2Sn4PunwpHCDHRLiK3DFsyFuXkrEn
NjWzYIJS42LdWgqaHQnFRvFRxarPpWueCUTgTOIjJoxBSv2S90g7psQpnKnk
jlLoZmppij18QEzxOTk.Q6kUytaUb80fb8zNL9tlxatcb1UFUiySGUjzYS11
GgJ6BjioBjn5v2vNEeOpLSrsW5G6MixePNt9LGj4o9U1UPRYT0gnmoPQtVbe
A.0tUpOaAi9MnnCirXoCjTuLAStDjk7XTgN1KqqOtLcFik0mUy5xv2KpXefP
oCzhB1goYVP1s+DqMlIYlep8VygusjZ3tU5TH1xQOzWaKPYYUYm6u8Ognjbj
.KHFSfsUCyixL.6y4gQ3jJcxSvORREljLccFjuN4PsSzpFqbJYGlK5SSHyf0
mBW7rQo2gTYcZysxZ2Gxjnn+KHCOHbAeO6Qd0KV6n0UAzNQhtA0cyX2i9IaG
qecOSaDsLFIoXk2qksI23vZamoqro5wdxlylpAsE0krpxv7JnLzsQMGcQHzD
PaekJC22YkQBKW1GqXTkwm2S3.caP.4C5xXo3T.gBPfZm5MfYoubqzSUsGMP
cojXulebpa9wQVNHR27iSfgFbvZG4RMS1CzTiR5MPW9wJEoJ1NE7g59K+vrz
dlaKXFSncvzJ.XT2sqWAuvlbbuSJmdccOVm2MWc.zsmavcUkfAp7Tf6TwTUu
.3N8MIMLjAt.48Y.p69.LWlAnuDCPckGf9VPf6jsWd.HqVsu64HiPwIrRpXX
T2zQ+lVU7sbpmI3Yh9WehL.1WfWq8aogw33V6wNKG1.ysgbrtxrivKP2LV1w
1ltTF6IJkpQih+3JMNqrHo9DWY3.8Alr6AAg1zB5WZRwCffuNKS1K8L3OyyP
0YcTUyxVJ2no7fe5.iiALdx+UhKdV1tsiKHJLv+D2f12LYceyvKgW3Tersde
lytIRwzWIHFwIImEpAAcmXvELfqKbJz58pebfA3C88pf+Q9bvdN9S4z615wc
jJdtRJXFBJ7FHm4.H07D67RMyR.9pKY2wEr0UI3AvYBLCuAJW2p7eyRRUuTN
I8.iPEUdnNtNa7jEZkEWNZTp199miWjYlNVN0e4JQj2ak+u+brQ12.AoU+1m
QRA2JA8ln6BmAhtYBx55busiTdl1Vle70fvi+1s1ENbFwjQ2.kSzLTN92hL3
yJc1QBxTKbvTSUxXvzRGLozimR5zSHc3zQ0SF0LnnA0ealu3egxY4ns6I6jP
e2dANcKqyOZV8PI6qRPIIx6d14VzPyb28Val.Or0mYckpsPtfgSrZUtbmI00
Wp0PSLD2k+Z4+CLX4uSI
-----------end_max5_patcher-----------
</code></pre>

**Comment by jcelerier** [comment]
yep, shouldn't be too hard to do also if someone else want to look into the code (@thibaudk ?)
